app/views/pages/index.html.erb

```diff
@@ -1,14 +1,55 @@
-<div class="container text-center">
-  <h1>Pages#index</h1>
-  <p>Find me in app/views/pages/index.html.erb</p>
+<div class="container">
   <% if current_user %>
-    <h2>Logged-in User:</h2>
+    <div class="row">
+      <div class="col col-lg-4">
+        <%= link_to 'Create Bet', new_bet_path %>
+      </div>
+      <div class="col col-lg-4">
+      </div>
+      <div class="col col-lg-3">
+      </div>
+      <div class="col col-lg-1">
+        <div class="dropdown">
+            <img class="" type="button" data-toggle="dropdown" src="http://fuuse.net/wp-content/uploads/2016/02/avatar-placeholder.png" style="width: 100%;">
```

Comment:
4 space indent and not sure we aren't using the Rails image_tag?
http://apidock.com/rails/ActionView/Helpers/AssetTagHelper/image_tag
Also noticing you have a class attr with no data. Not sure if needed?